Cacher.conf

```diff
@@ -1,12 +1,12 @@
-Threads=2
-StatsPPS=true
-StatsbPS=true
-GameUpdateInterval=1000
-GameUpdateSocketTimeout=1000
-ChallengeCodeTTL=5000
-LocalServerIPAddress=0.0.0.0
-LocalServerPort=9110
-GameServerIPAddress=192.168.1.2
-GameServerPort=27015
-ReceiveBufferSize=65535
-SendBufferSize=65535
+Threads=1
+StatsPPS=true
+StatsbPS=true
+GameUpdateRate=1000
+GameUpdateTimeout=1000
+ChallengeCodeTTL=5000
+LocalServerIPAddress=127.0.0.1
+LocalServerPort=43001
+GameServerIPAddress=88.198.1.79
```

Comment:
GameServerIPAddress sould probably be reset to 127.0.0.1 or whatever you use in your tests